package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "convict": "0.6.1",
     "hapi": "8.2.0",
+    "hapi-fxa-oauth": "2.0.1",
```

Comment:
We should consider moving this repo under "mozilla" org, adding additional maintainers on npm, etc.